### ISSUE TITLE
Using paste in the Pin code input causes issues

### ISSUE THREAD

**Issue by TNXG** [themesberg/flowbite#954]
**Describe the bug**
When using the Pin code input feature within a number-input component, pasting a verification code only pastes the character into the first input field, while the remaining digits are omitted. This behavior can significantly affect the user experience, as typically, pasting a verification code should fill all input fields.

**To Reproduce**

It seems that this issue is also present in the official documentation for this feature: [number-input/#pin-code-input](https://flowbite.com/docs/forms/number-input/#pin-code-input)

**Expected behavior**

When pasting a multi-digit verification code, it should automatically fill in the corresponding fields.

**Screenshots**

![recording](https://github.com/user-attachments/assets/c886ffbc-2967-49e0-8840-ae9eed018900)

**Desktop (please complete the following information):**
```text
System:
    OS: Windows 11 10.0.22631
    CPU: (6) x64 Intel(R) Core(TM) i5-9600KF CPU @ 3.70GHz
    Memory: 11.32 GB / 23.94 GB
  Binaries:
    Node: 20.11.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.22 - C:\Program Files\nodejs\yarn.CMD
    npm: 10.2.4 - C:\Program Files\nodejs\npm.CMD
    pnpm: 9.9.0 - ~\AppData\Local\pnpm\pnpm.EXE
  Browsers:
    Edge: Chromium (127.0.2651.74)
    Internet Explorer: 11.0.22621.3527
  npmPackages:
    @nuxt/kit: ^3.13.0 => 3.13.0
    flowbite: ^2.5.1 => 2.5.1
    nuxt: ^3.13.0 => 3.13.0
    vite: ^5.4.2 => 5.4.2
    vue: ^3.4.38 => 3.4.38
```

**Comment by LLLongA** [comment]
I fix this issue and start a pull request in #979.

**Comment by TNXG** [comment]
Considering that the issue has been resolved by the relevant PR and has been merged, this issue will be closed.
